react/ConditionLayoutCategory.tsx

```diff
@@ -0,0 +1,78 @@
+import React, { useMemo } from 'react'
+import type { ComponentType } from 'react'
+import { useRuntime } from 'vtex.render-runtime'
+
+import ConditionLayout from './ConditionLayout'
+import type { NoUndefinedField, MatchType, Condition, Handlers } from './types'
+
+type Props = {
```

Comment:
Change to interfaces, because it is better.
```suggestion
interface Props {
```